### DISCUSSION TITLE
Cannot Set RefindPlus to Always Select the Last Booted Loader

### DISCUSSION THREAD

**Discussion by volcbs** [dakanji/RefindPlus#66] [General]
I honesty don't know where to put this thread, but can you tell me what's wrong with my config? 

I'm trying to make RefindPlus picking my previous selection for the next boot, but it keeps booting the first choice. I already enable use_nvram and set default_selection to +, I also tried with just set default_selection to + and it still doesn't work.

[config.conf.zip](https://github.com/dakanji/RefindPlus/files/6910841/config.conf.zip)

**Comment by dakanji** [comment]
This is the right place for queries such as yours.

RefindPlus keeps selecting one option because you have set the `default_selection` token. By setting this, you are telling it to ALWAYS select a certain item. 

If you don't want it to do this, and want it to select the previously booted option each time instead, you should deactivate the token by deleting or commenting it out as explained in the notes that come with the sample config file:
https://github.com/dakanji/RefindPlus/blob/6be94e202a468c39d29ad67b075138401f2e128b/config.conf-sample#L853-L861 

**EDIT**
Missed that it was set to "+" which should also work for default selection.
Either way, seems most likely setting was been overridden subsequently

**Reply by volcbs** [reply]
Oh you're right.. I thought it needs to be _defined_ to be like that. Thanks

**Reply by volcbs** [reply]
@dakanji 
Sorry, I was really r*tarded to notice that I actually accessed my windows from opencore instead of the actual windows bootloader. I already do like you say and it still doesn't work. 
[config.conf.zip](https://github.com/dakanji/RefindPlus/files/6915982/config.conf.zip)
How do I create logs for this?

**Reply by dakanji** [reply]
I don't understand. RefindPlus does not control what happens in OpenCore.

**Reply by volcbs** [reply]
Yeah, I know. I thought it fixed the problem (your answer/solution), but it turned out it didn't. What I thought was fixed is refindplus picked opencore that somehow picked windows boot instead of mac boot. So, the problem where refindplus picks the first choice instead of previous boot still occurs.
Sorry if I make you misunderstand the problem.

**Reply by dakanji** [reply]
RefindPlus always defaults to loading the previously booted item unless overridden.

While you may have removed the setting overriding this in config.conf, are you sure the included theme conf file does not contain a setting that overrides this default?

If you are certain, please raise an issue and provide the information specified in the issue template.
